src/cmd/append.go

```diff
@@ -81,10 +85,13 @@ func getAppendConfig(args []string, repo *git.ProdRepo) (result appendConfig, er
 		return result, err
 	}
 	result.ancestorBranches = git.Config().GetAncestorBranches(result.parentBranch)
-	result.hasOrigin = git.HasRemote("origin")
+	result.hasOrigin, err = repo.Silent.HasRemote("origin")
```

Comment:
`repo.Silent.HasRemote("origin")` is called twice in this function. Thoughts on updating the first call to set `result.hasOrigin`?

Reply:
Nice catch!